epoxy-pagingsample/src/main/java/com/airbnb/epoxy/pagingsample/PagingSampleActivity.kt

```diff
@@ -0,0 +1,96 @@
+package com.airbnb.epoxy.pagingsample
+
+import android.arch.paging.*
+import android.content.*
+import android.os.*
+import android.support.v7.app.*
+import android.support.v7.widget.*
+import android.widget.*
+import com.airbnb.epoxy.*
+import com.airbnb.epoxy.paging.*
+import android.arch.persistence.room.*;
+import kotlinx.coroutines.experimental.*
+import kotlinx.coroutines.experimental.android.*
+import org.jetbrains.anko.coroutines.experimental.*
+import java.util.concurrent.*
+
+class PagingSampleActivity : AppCompatActivity() {
+
+    lateinit var db: PagingDatabase
+
+    public override fun onCreate(savedInstanceState: Bundle?) {
+        super.onCreate(savedInstanceState)
+        setContentView(R.layout.activity_main)
+        db = Room.databaseBuilder(applicationContext,
+                                  PagingDatabase::class.java,
+                                  "database-name").build()
+
+        val pagingController = TestController()
+        val recyclerView = findViewById<RecyclerView>(R.id.recycler_view)
+        recyclerView.layoutManager = LinearLayoutManager(this)
+        recyclerView.adapter = pagingController.adapter
+
+        async(UI) {
+            val pagedList = bg {
+                db.userDao().delete(db.userDao().all)
+                for (i in 1..3000) {
+                    db.userDao().insertAll(User(i))
+                }
+
+                return@bg PagedList.Builder<Int, User>().run {
+                    setDataSource(db.userDao().dataSource)
+                    setMainThreadExecutor(UiThreadExecutor)
+                    setBackgroundThreadExecutor(AsyncTask.THREAD_POOL_EXECUTOR)
+                    setConfig(PagedList.Config.Builder().run {
+                        setEnablePlaceholders(false)
+                        setPageSize(40)
+                        setInitialLoadSizeHint(80)
```

Comment:
Accord to the [doc](https://developer.android.com/reference/android/arch/paging/PagedList.Config.Builder.html#setInitialLoadSizeHint(int)), this will be ignored currently when using TiledDataSource.

`Defines how many items to load when first load occurs, if you are using a KeyedDataSource.
If you are using an TiledDataSource, this value is currently ignored`